ext/AvizExt/viz/viz.jl

```diff
@@ -81,6 +81,18 @@ function _calc_gridsize(n_factors::Int64; max_cols::Int64=4)::Tuple{Int64,Int64}
     return n_rows, n_cols
 end
 
+function confint(
+    data::NamedDimsArray, dimension::Symbol
+)::Tuple{Vector{Float64},Vector{Float64},Vector{Float64}}
+    data_slices = Slices(data, NamedDims.dim(data, dimension))
+
+    y_lower = quantile.(data_slices, [0.025])
+    y_upper = quantile.(data_slices, [0.975])
+    y_median = median.(data_slices)
```

Comment:
That doesn't work (I had tried that before). What DOES "work" (and I just found out that) is:

```julia
julia> quantile.(data_slices, [[0.025] [0.5] [0.975]])
75×3 Matrix{Float64}:
 6.58232e6  6.58232e6  6.58232e6
 2.79034e6  5.56852e6  6.16467e6
 2.33228e6  4.39754e6  5.80106e6
 2.31361e6  3.74498e6  5.34082e6
 ⋮
 2.51627e5  3.5774e5   5.34254e5
 2.36002e5  3.63243e5  5.50408e5
 2.08632e5  3.53628e5  5.27182e5
```

I just updated this function now, see what you think.

Reply:
@ConnectedSystems 
Another question: I think this function should have another name and live somewhere else. I was thinking about `confint_series` or `series_confint` because it computes confints of multiple series (it is different from a common "confint" function that receives one vector of data). And also it may be useful outside visualization, so I was thinking maybe moving somewhere inside Analysis module. What do you think? Where would be a good place for it?

Reply:
The `analysis` module seems like the best place for it...